hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/table/HoodieFlinkTable.java

```diff
@@ -114,4 +114,11 @@ private static void setLatestInternalSchema(HoodieWriteConfig config, HoodieTabl
       config.setInternalSchemaString(SerDeHelper.toJson(internalSchema.get()));
     }
   }
+
+  /**
+   * Flink writer need to skip check the compatibility of Schema#name.
+   */
+  public boolean isNeedCheckSchemaNames() {
```

Comment:
Pls add `@Override`

Reply:
thank you for review.

Reply:
Only flink ignores that namespace name or all the engines can do that?

Reply:
only Flink needs to ignore namespace name.

Reply:
IsNeedCheckSchemaNames returns true by default, so all engines need check by default.